core/storage/statistics/gae_models.py

```diff
@@ -696,6 +893,21 @@ def _shard_answers(
         list, shard the answers such that a list of answer lists are returned.
         The first entry is guaranteed to contain all answers of the current
         answer list.
+
+        Args:
+            current_answer_list: list(Json). list of answer dicts each of which
+            is stored as Json blob.
+            current_answer_list_size: list(int). Number of bytes required
+            to store allthe answers in the current_answer_list.
+            new_answer_list:list(Json). List of new submitted answers each of
```

Comment:
Missing space after colon, here and elsewhere.

Reply:
done